bindings/python/setup.py

```diff
@@ -29,22 +29,22 @@
 def target(arch):
     return cpu.TARGETS[arch]
 
-# N.B.: cibuildwheel must configure the multi-arch environment variable.
-# Also, the micro-architectures defined below should be in order of preference.
-if os.environ.get("SVS_MULTIARCH", None) is not None:
-    svs_microarchs = [
+# TODO: Replace with externally-specified list
+svs_microarchs = [
+        # "x86_64_v3" # This is the default target for base lib compilation
+        "broadwell",
+        "skylake",
+        "skylake_avx512",
         "cascadelake",
-        "x86_64_v3", # conservative base CPU for x86 CPUs.
+        # TODO: Add support for other architectures (archspec does not support them yet)
+        # "cooperlake",
+        # "icelake_server",
+        "sapphirerapids",
+        # "graniterapids",
+        # "graniterapids_d",
```

Comment:
@napetrov I remember we had a discussion earlier on how many microarchs to build the Python package with as the time/size increase with each new entry here. Do we need all of them? Also, we need to keep them in sync with the private repository for building the shared library

Reply:
Agree that we should be careful with adding those - we not necessary need to build entire library for codepath to utilize some of the features.  

We can call for bf16 even from sse codepath - the only thing that compilation for particular target would do is compiler code optimization and i doubt that there is perf difference for generic code despite specific functionality usage -this is usually works out to be just run. 

Also we would need older isa set - prior to broadwell for clean run on systems that doesn't support AVX2